package.json

```diff
@@ -104,19 +103,18 @@
   },
   "scripts": {
     "prebuild": "node bin/fetch_api_docs_and_roadie_local.js",
-    "build": "yarn clean && yarn prebuild && gatsby build",
+    "build": "yarn prebuild && gatsby build",
```

Comment:
Using clean is now a bad practice: https://www.gatsbyjs.com/docs/how-to/performance/improving-build-performance/#make-sure-youre-not-clearing-the-cache-between-builds